experimental/learn_descriptors/four_seasons_parser.cc

```diff
@@ -42,17 +48,17 @@ FourSeasonsParser::FourSeasonsParser(const std::filesystem::path& root_dir,
     txt_parser_help::TimeDataMap vio_poses_time_map =
         txt_parser_help::create_vio_time_data_map(path_vio, min_time_sig_figs);
     gps_parser_help::TimeGPSList gps_time_list =
-        gps_parser_help::create_gps_time_data_map(path_gps);
+        gps_parser_help::create_gps_time_data_list(path_gps);
 
-    const double imu_hz = 30.0;
     size_t id = 0;
-    size_t gps_idx = 0;
+    std::vector<size_t> img_times;
     for (const std::pair<size_t, std::vector<std::string>>& pair_time_data : img_time_list) {
         const size_t time_key = pair_time_data.first;
         ImagePoint img_pt;
         img_pt.id = id;
         img_pt.seq = std::stoull(
             pair_time_data.second[static_cast<size_t>(txt_parser_help::ImgIdx::TIME_NS)]);
+        img_times.push_back(img_pt.seq);
```

Comment:
It seems unnecessary to create a copy of the data here since you already have a list of image points